### PR TITLE
added popup created by Tetiana D.

### PR DESCRIPTION
**Pull request by TetianaDiachenko**
### Hooli-style Popup

[Demo](https://tetianadiachenko.github.io/Popup/) |
[Code base](https://github.com/TetianaDiachenko/Popup)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by TetianaDiachenko** [comment]
I checked my work. I hope, there are no common mistakes in it. Please review my popup.

File: submissions/TetianaDiachenko/popup_td/index.html [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header">
+        <div class="wrapper">
+            <div class="header__content">
+                <div class="header__logo">
+                    <a href="https://kottans.org/" target="_blank">
+                        <img src="./img/kottans.png" alt="kottans">
+                    </a>
```
Comment (A-Ostrovnyy) [inline submissions/TetianaDiachenko/popup_td/index.html:20]: :fire:

File: submissions/TetianaDiachenko/popup_td/index.html [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+                </div>
+                <nav class="header__nav">
+                    <ul class="header__list" >
+                        <li class="header__item">
+                            <a href="#">Mail</a>
```
Comment (A-Ostrovnyy) [inline submissions/TetianaDiachenko/popup_td/index.html:25]: Avoid elements shifting on hover or focus stgate. [Watch video](https://share.vidyard.com/watch/pPNgFXoNLxepcK7MKUbWKx?)

File: submissions/TetianaDiachenko/popup_td/index.html [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+                                    <li>
+                                        <a href="#">
+                                            <img src="./img/searchi.png" alt="">
+                                            <p>Search</p> 
+                                        </a>
+                                    </li>                                    
+                                </ul>
+                                <div class="popup__more">
+                                    <input type="checkbox" name="more" id="more" class="more__check">
+                                    <label for="more" class="popup__mbtn">More</label>
```
Comment (A-Ostrovnyy) [inline submissions/TetianaDiachenko/popup_td/index.html:94]: Logically hide this button when the second part of the popup is open

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Nice design!

File: submissions/TetianaDiachenko/popup_td/index.html (hunk 2 above)
Comment (TetianaDiachenko) [reply submissions/TetianaDiachenko/popup_td/index.html:25]: Thank you for you answer. 
If you could please say how to do this? or what to read/watch how to do this?

File: submissions/TetianaDiachenko/popup_td/index.html (hunk 3 above)
Comment (TetianaDiachenko) [reply submissions/TetianaDiachenko/popup_td/index.html:94]: Sorry, but I don't agree with you. I left this button to have a possibility to close the second part of the pop-up.

File: submissions/TetianaDiachenko/popup_td/index.html (hunk 2 above)
Comment (A-Ostrovnyy) [reply submissions/TetianaDiachenko/popup_td/index.html:25]: 1. You can use an outline or border
2. By default you can set a transparent border and just change border color when you need it
Also, I'm right now streaming in discord so  you can ask me dirrectly)

**Comment by TetianaDiachenko** [comment]
Thank you! I fixed the bugs and sent changes.

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**